fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -3328,24 +3328,43 @@ void DDTeamCollection::updateCpuPivots() {
 }
 
 void DDTeamCollection::updateTeamEligibility() {
+	int healthyCount = 0, lowDiskUtilTotal = 0, lowCpuTotal = 0, lowDiskUtilLowCpu = 0;
```

Comment:
```suggestion
	int healthyCount = 0, lowDiskUtilTotal = 0, lowCpuTotal = 0, allTypeLow = 0;
```
In order to not modify the name if we add more eligibility type.

Reply:
Changed to 'allMetricsLow'